src/pages/home/report/ReportActionCompose.js

```diff
@@ -289,6 +338,44 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
+                    <PopoverWithMeasuredContent
+                        isVisible={this.state.isPickerVisible}
+                        onClose={this.hideEmojiPicker}
+                        anchorPosition={this.popoverAnchorPosition}
+                        animationIn="fadeIn"
+                        anchorOrigin={
+                            {
+                                horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                                vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                            }
+                        }
+                        measureContent={() => (
+                            <EmojiPickerMenu
+                                isVisible
+                                addEmojiToTextBox={this.addEmojiToTextBox}
+                            />
+                        )}
+                    >
+                        <EmojiPickerMenu
+                            isVisible={this.state.isPickerVisible}
```

Comment:
Mentioned above, but since this is in a modal anyways, I think we might be able to just get rid of the `isVisible` prop and simplify things. That would also mean that the children and the `measureContent` would be exactly the same, which is why I would be happy to see that refactored.